README.md

```diff
@@ -60,6 +60,14 @@ I concluded that a solution must meet the following principles:
 
 With these principles in mind, I went out and implemented the React Page, which you are looking at right now.
 
+### Bundle Size
+
+We try to keep the initial bundle size low so that you can use this library also to render the content statically without edit functionality.
+
+We achieve that by lazy-loading using `import()` functions. Most modern bundlers like webpack (e.g. in nextjs) support this kind of lazy loading. So the default editor-ui is only loaded if the editor is not in editMode.
```

Comment:
`So the default editor-ui is only loaded if the editor is not in editMode.` I think it's the opposite here.

Reply:
good catch ;-) will fix